source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -0,0 +1,437 @@
+feature that reports failures and crashes to the |CL| development
+team for improvements. For more information, see :ref:`telemetry-about`.
+
+#. From :guilabel:`Required Options`, select :guilabel:`Telemetry`.
+
+#. Select :kbd:`Confirm`.
+
+#. If you don't wish to participate, deselect :kbd:`Enable Telemetry`.
+
+#. Select :kbd:`Confirm`.
+
+   .. figure:: figures/bare-metal-install-desktop-11.png
+      :scale: 100%
+      :alt: Enable Telemetry
+
+      Figure 11: Enable Telemetry
+
+Advanced options
+****************
+
+After you complete the `Required options`_, we recommend completing
+:guilabel:`Advanced options`--though they're not required. Doing so
+customizes your development environment, so you're ready to go immediately
+after reboot.
+
+* `Bundle Selection`_
+
+* `Assign Hostname`_
+
+As for bundles, you can always add more later with :ref:`swupd-guide`.
+
+Bundle Selection
+================
+
+#. On the Advanced menu, select :guilabel:`Bundle Selection`
+
+#. Select your desired bundles.
+
+   .. figure:: figures/bare-metal-install-desktop-12.png
+      :scale: 100%
+      :alt: Bundle Selection
+
+      Figure 12: Bundle Selection
+
+#. Select :kbd:`Confirm`.
+
+#. View the bundles that you selected.
+
+   .. figure:: figures/bare-metal-install-desktop-13.png
+      :scale: 100%
+      :alt: Bundle Selections - Advanced Options
+
+      Figure 13: Bundle Selections - Advanced Options
+
+Optional: Skip to `Finish installation`_.
+
+Assign Hostname
+===============
+
+#. In Advanced Options, select :guilabel:`Assign Hostname`.
+
+#. In :guilabel:`Hostname`, enter the hostname only (excluding the domain).
+
+   .. figure:: figures/bare-metal-install-desktop-14.png
+      :scale: 100%
+      :alt: Assign Hostname
+
+      Figure 14: Assign Hostname
+
+   .. note::
+
+      Hostname does not allow empty spaces. Hostname must start with an
+      alphanumeric character but may also contain hyphens. Maximum length of
+      63 characters.
+
+#. Navigate to :kbd:`Confirm` until highlighted.
+
+#. Select :kbd:`Confirm`.
+
+Optional: Skip to `Finish installation`_.
+
+Finish installation
+*******************
+
+#. When you are satisfied with your installation configuration, select
+   :guilabel:`Install`.
+
+   .. figure:: figures/bare-metal-install-desktop-15.png
+      :scale: 100%
+      :alt: Assign Hostname
+
+      Figure 15: Finish installation
+
+   .. note:
+
+      All checkmarks must be visible in :guilabel:`Required Options` for you to select :guilabel:`Install`.
+
+#. If you do not enter a selection for all :guilabel:`Required Options`,
+   the :guilabel:`Install` button remains greyed out, as shown
+   in Figure 16. Return to `Required Options`_ and make selections.
+
+
+   .. figure:: figures/bare-metal-install-desktop-16.png
+      :scale: 100%
+      :alt: Required Options - Incomplete
+
+      Figure 16: Required Options - Incomplete
+
+#. Select :guilabel:`reboot`.
```

Comment:
Actually, the GUI diverged from the TUI for the end of the Install.
The only button option is `Exit`
Then the user would need to select Reboot option from the Gnome Desktop.

Reply:
See my latest commit. Will you please clarify:
 
- After pressing `Install`, does target system automatically reboot; or 
- After pressing `Install`, must user "select Reboot option from the Gnome Desktop."
  FYI: I didn't have enough free space on either box to go this far. 

I added `Exit` as `Optional` step.

Reply:
After Install, when the installation completes, the user must press Exit to close the installer. That should be the standard flow, not really optional. 

After closing the Installer, the user can then reboot by using the standard Gnome desktop menu option to shutdown or reboot. Otherwise they are still running the live image.

Reply:
Yep. Thanks.